contrib/IECoreUSD/test/IECoreUSD/USDSceneTest.py

```diff
@@ -4257,5 +4259,28 @@ def testPerPurposeModelBound( self ) :
 		self.assertTrue( root.child( "group" ).hasBound() )
 		self.assertEqual( root.child( "group" ).readBound( 0 ), imath.Box3d( imath.V3d( -1 ), imath.V3d( 1 ) ) )
 
+	def testAsetPathSlashes ( self ) :
```

Comment:
Aset -> Asset

Reply:
Fixed in https://github.com/ImageEngine/cortex/pull/1426/commits/a3c9fff781461093087c019f629421730e65125b